manalize-tmp.phar

```diff
@@ -0,0 +1 @@
+<html><body>You are being <a href="https://github-cloud.s3.amazonaws.com/releases/62733701/9e3dbb38-80db-11e6-8847-641ac8c7a8db.phar?X-Amz-Algorithm=AWS4-HMAC-SHA256&amp;X-Amz-Credential=AKIAISTNZFOVBIJMK3TQ%2F20161009%2Fus-east-1%2Fs3%2Faws4_request&amp;X-Amz-Date=20161009T120158Z&amp;X-Amz-Expires=300&amp;X-Amz-Signature=6ab8dcc72de59515c8760f7f09719afdc2300fca243f1d9ab4191f0f78ecbd99&amp;X-Amz-SignedHeaders=host&amp;actor_id=0&amp;response-content-disposition=attachment%3B%20filename%3Dmanalize.phar&amp;response-content-type=application%2Foctet-stream">redirected</a>.</body></html>
```

Comment:
I can't understand how this happened. Maybe we should add a check in the `make build` for removing this file if exists